programs/server/config.xml

```diff
@@ -713,31 +713,31 @@
              For example, if there two users A, B and a row policy is defined only for A, then
              if this setting is true the user B will see all rows, and if this setting is false the user B will see no rows.
              By default this setting is false for compatibility with earlier access configurations. -->
-        <users_without_row_policies_can_read_rows>false</users_without_row_policies_can_read_rows>
```

Comment:
Maybe it wroth to change the default values of the getters as well, i.e.:

```
$ gg 'getBool("access_control_improvements'
src/Access/AccessControl.cpp:    setEnabledUsersWithoutRowPoliciesCanReadRows(config_.getBool("access_control_improvements.users_without_row_policies_can_read_rows", false));
src/Access/AccessControl.cpp:    setOnClusterQueriesRequireClusterGrant(config_.getBool("access_control_improvements.on_cluster_queries_require_cluster_grant", false));
src/Access/AccessControl.cpp:    setSelectFromSystemDatabaseRequiresGrant(config_.getBool("access_control_improvements.select_from_system_db_requires_grant", false));
src/Access/AccessControl.cpp:    setSelectFromInformationSchemaRequiresGrant(config_.getBool("access_control_improvements.select_from_information_schema_requires_grant", false));
src/Access/AccessControl.cpp:    setSettingsConstraintsReplacePrevious(config_.getBool("access_control_improvements.settings_constraints_replace_previous", false));

```

Reply:
Yes, it makes sense. Please send a PR.

Reply:
https://github.com/ClickHouse/ClickHouse/pull/60153